log/log.go

```diff
@@ -132,9 +135,17 @@ func StartObserver(o Options) error {
 
 	// create a client
 	logClient := NewClient(gRPC, o.MsgPath, o.LogPath, o.LogFilter, o.Limit)
-	if logClient == nil {
-		return errors.New("failed to connect to the gRPC server\nPossible troubleshooting:\n- Check if Kubearmor is running\n- Create a portforward to KubeArmor relay service using\n\t\033[1mkubectl -n kube-system port-forward service/kubearmor --address 0.0.0.0 --address :: 32767:32767\033[0m\n- Configure grpc server information using\n\t\033[1mkarmor log --grpc <info>\033[0m")
+	for logClient == nil {
+		pf, err := utils.InitiatePortForward(c, 32767, 32767, matchLabels)
```

Comment:
With this logic, if someone tries to execute `karmor log` from multiple terminals ... It will still fail since the local port 32767 will be busy.

We should keep this logic in a for loop for maybe 10/20 ports starting 32767.

```
// Pseudo code
NewClient() // default New Client
for lport = 32767; lport < 32767 + 20 ; lport++  {
     err = InitiatePortForward(c, serverPort, lport, matchLabels)
     if err == nil {
          NewClient()
          break
     }
}
```

Reply:
Question is, if it failed outside of for loop .. it will also fail continuously within the for loop as well since the same ports are used.